tdrs-backend/tdpservice/users/migrations/0015_replace_data_analyst_role.py

```diff
@@ -0,0 +1,21 @@
+
+# Generated by Django 3.1.4 on 2021-01-08 17:41
+
+from django.contrib.auth.models import Group
+from django.db import migrations
+
+
+def change_user_groups(apps, schema_editor):
```

Comment:
We reuse the original entry for Data Prepper, just changing its name, so the users are all still connected to their original group.